### PR TITLE
Channel edits: Further sanitizing us.m3u playlist

### PR DESCRIPTION
**Pull request by jkemp814**
The work continues.

### CODE REVIEW

**Comment by jkemp814** [comment]
Can someone take a look at the tvg-id field at line 2481?  Midnight Pulp. Then advise the best course of action.

**Comment by LaneSh4d0w** [comment]
> 
> 
> Can someone take a look at the tvg-id field at line 2481? Midnight Pulp. Then advise the best course of action.

#EXTINF:-1 tvg-id="MidnightPulp.us" tvg-name="Midnight Pulp" tvg-country="US" tvg-language="English" tvg-logo="https://od.lk/s/MF8yMjQwMDkyNjVf/MidnightPulp_200x200.png" group-title="Entertainment",Midnight Pulp
https://369f2966f62841f4affe37d0b330a13c.mediatailor.us-east-1.amazonaws.com/v1/master/44f73ba4d03e9607dcd9bebdcb8494d86964f1d8/Plex_MidnightPulp/playlist.m3u8?ads.wurl_channel=263&ads.wurl_name=MidnightPulp&ads.us_privacy=1---&ads.psid=&ads.targetopt=1&ads.plex_token=z1MCPUpbxYcHru-5hdyq&ads.plex_id=5ef4e1b40d9ad000423c4427&ads.ua=Mozilla%2F5.0+%28Windows+NT+10.0%3B+Win64%3B+x64%29+AppleWebKit%2F537.36+%28KHTML%2C+like+Gecko%29+Chrome%2F84.0.4147.89+Safari%2F537.36+OPR%2F70.0.3728.71&ads.app_bundle=&ads.app_store_url=&ads.gdpr=0&ads.consent=0
#EXTINF:-1 tvg-id="MidnightPulp.us" tvg-name="Midnight Pulp" tvg-country="US" tvg-language="English" tvg-logo="https://od.lk/s/MF8yMjQwMDkyNjVf/MidnightPulp_200x200.png" group-title="Entertainment",Midnight Pulp (1080p)
https://midnightpulp-samsung.plex.wurl.com/manifest/playlist.m3u8
#EXTINF:-1 tvg-id="MidnightPulp.us" tvg-name="Midnight Pulp" tvg-country="US" tvg-language="English" tvg-logo="https://od.lk/s/MF8yMjQwMDkyNjVf/MidnightPulp_200x200.png" group-title="Entertainment",Midnight Pulp
https://midnightpulp-samsung.roku.wurl.com/manifest/1200.m3u8
#EXTINF:-1 tvg-id="MidnightPulp.us" tvg-name="Midnight Pulp" tvg-country="US" tvg-language="English" tvg-logo="https://od.lk/s/MF8yMjQwMDkyNjVf/MidnightPulp_200x200.png" group-title="Entertainment",Midnight Pulp
https://midnightpulp-samsung.vizio.wurl.com/manifest/playlist.m3u8``

**Comment by LaneSh4d0w** [comment]
> 
> 
> Can someone take a look at the tvg-id field at line 2481? Midnight Pulp. Then advise the best course of action.

2481 should be 
#EXTINF:-1 tvg-id="MidnightPulp.us" tvg-name="Midnight Pulp" tvg-country="US" tvg-language="English" tvg-logo="https://od.lk/s/MF8yMjQwMDkyNjVf/MidnightPulp_200x200.png" group-title="Entertainment",Midnight Pulp

**Comment by Dum4G** [comment]
@LaneSh4d0w can you propose a quick fix? I'm sharing it as a full view gist

**Comment by LaneSh4d0w** [comment]
Done

File: channels/us.m3u [hunk 1]
```diff
@@ -3048,8 +3032,8 @@ https://a357e37df8ec46719fdeffa29a3e8e40.mediatailor.us-east-1.amazonaws.com/v1/
 #EXTINF:-1 tvg-id="PeopleareAwesome.us" tvg-name="People are Awesome" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/msHBlDz.png" group-title="Entertainment",People are Awesome
 https://dai2.xumo.com/amagi_hls_data_xumo1212A-peopleareawesome/CDN/playlist.m3u8
 #EXTINF:-1 tvg-id="PeopleareAwesome.us" tvg-name="People are Awesome" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/msHBlDz.png" group-title="Entertainment",People are Awesome
-https://dai2.xumo.com/amagi_hls_data_xumo1212A-redboxpeopleareawesome/CDN/playlist.m3u8?p=redbox&deviceid=&is_lat=&subp=RedboxdesktopWebWindows
-#EXTINF:-1 tvg-id="PeopleareAwesome.us" tvg-name="People are Awesome" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/msHBlDz.png" group-title="Entertainment",People are Awesome
+https://dai2.xumo.com/amagi_hls_data_xumo1212A-redboxpeopleareawesome/CDN/playlist.m3u8
+#EXTINF:-1 tvg-id="PeopleareAwesome.us" tvg-name="People_are_Awesome" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/msHBlDz.png" group-title="Entertainment",People are Awesome
```
Comment (Dum4G) [inline channels/us.m3u]: Underscores

File: channels/us.m3u [hunk 2]
```diff
@@ -2386,12 +2380,6 @@ https://mhz-samsung-linear-ca.samsung.wurl.com/manifest/playlist.m3u8
 https://mhz-samsung-linear.samsung.wurl.com/manifest/playlist.m3u8
 #EXTINF:-1 tvg-id="MiamiCityTV.us" tvg-name="Miami City TV" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/HJohrzO.png" group-title="Local",Miami City TV (360p)
 https://granicusliveus9-a.akamaihd.net/miamifl/G2076_003/playlist.m3u8
-#EXTINF:-1 tvg-id="MiamiTV.us" tvg-name="Miami TV" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/WlrcyMe.png" group-title="XXX",Miami TV
-https://59ec5453559f0.streamlock.net/miamitv/smil:miamitvROKU/chunklist_w1195980091_b2592000.m3u8
```
Comment (Dum4G) [inline channels/us.m3u]: miamitvROKU, can be alternate variant

**Review by Dum4G — CHANGES_REQUESTED** [review]

**Comment by LaneSh4d0w** [comment]
There's one "People are Awesome" that has got some underlines.

File: channels/us.m3u (hunk 2 above)
Comment (LaneSh4d0w) [reply channels/us.m3u]: Yup, there's a playlist below it.

File: channels/us.m3u (hunk 1 above)
Comment (jkemp814) [reply channels/us.m3u:3036]: Good catch

File: channels/us.m3u (hunk 2 above)
Comment (Dum4G) [reply channels/us.m3u]: that is miamitvWEB

Comment (LaneSh4d0w) [reply channels/us.m3u]: Welp, my bad. Got up with the mexican link that had "Roku" in his name.

Comment (LaneSh4d0w) [reply channels/us.m3u:2390]: Fast patched up on his PR.

**Comment by jkemp814** [comment]
So, can this be moved to ready to review?

File: channels/us.m3u [hunk 3]
```diff
@@ -706,7 +706,7 @@ http://dai.google.com/linear/hls/event/rtcMlf4RTvOEkaudeany5w/master.m3u8
 https://cbsn-ny-cedexis.cbsnstream.cbsnews.com/out/v1/ec3897d58a9b45129a77d67aa247d136/master.m3u8
 #EXTINF:-1 tvg-id="CBSNNewYorkPlutoTV.us" tvg-name="CBSN New York (Pluto TV)" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/ubsNr5F.jpg" group-title="Local",CBSN New York (Pluto TV)
 https://service-stitcher.clusters.pluto.tv/stitch/hls/channel/5dc48170e280c80009a861ab/master.m3u8?terminate=false&deviceType=web&deviceMake=web&deviceModel=web&sid=206&deviceId=5dc48170e280c80009a861ab&deviceVersion=DNT&appVersion=DNT&deviceDNT=0&userId=&advertisingId=&deviceLat=&deviceLon=&app_name=&appName=web&buildVersion=&appStoreUrl=&architecture=&includeExtendedEvents=false&marketingRegion=US&serverSideAds=false
-#EXTINF:-1 tvg-id="CBSNewsBoston.us" tvg-name="CBSNews Boston" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/a2Qjf7T.png" group-title="Local",CBSNews Boston
+#EXTINF:-1 tvg-id="CBSNewsBoston.us" tvg-name="CBS News Boston" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/a2Qjf7T.png" group-title="Local",CBS News Boston
```
Comment (LaneSh4d0w) [inline channels/us.m3u:709]: I've seen you have did this. I have also began to rename the ones above accordingly, but didn't finish. Might be useful.

File: channels/us.m3u [hunk 4]
```diff
@@ -3119,19 +3129,19 @@ https://playerstv-samsungus.amagi.tv/playlist.m3u8
...
 https://service-stitcher.clusters.pluto.tv/v1/stitch/embed/hls/channel/5d4af00df345adb154b7a1f4/master.m3u8?deviceId=channel&deviceModel=web&deviceVersion=1.0&appVersion=1.0&deviceType=rokuChannel&deviceMake=rokuChannel&deviceDNT=1&advertisingId=channel&embedPartner=rokuChannel&appName=rokuchannel&is_lat=1&bmodel=bm1&content=channel&platform=web&tags=ROKU_CONTENT_TAGS&coppa=false&content_type=livefeed&rdid=channel&genre=ROKU_ADS_CONTENT_GENRE&content_rating=ROKU_ADS_CONTENT_RATING&studio_id=viacom&channel_id=channel
-#EXTINF:-1 tvg-id="70sCinema.us" tvg-name="70s Cinema" tvg-country="US" tvg-language="English" tvg-logo="http://images.pluto.tv/channels/5f4d878d3d19b30007d2e782/colorLogoPNG.png" group-title="Movies",Pluto TV 70s Cinema
+#EXTINF:-1 tvg-id="PlutoTV70sCinema.us" tvg-name="70s Cinema" tvg-country="US" tvg-language="English" tvg-logo="http://images.pluto.tv/channels/5f4d878d3d19b30007d2e782/colorLogoPNG.png" group-title="Movies",Pluto TV 70s Cinema
```
Comment (LaneSh4d0w) [inline channels/us.m3u:3138]: Incoherency in tvg-name ?

File: channels/us.m3u [hunk 5]
```diff
@@ -3119,19 +3129,19 @@ https://playerstv-samsungus.amagi.tv/playlist.m3u8
...
 https://service-stitcher.clusters.pluto.tv/v1/stitch/embed/hls/channel/5f4d878d3d19b30007d2e782/master.m3u8?deviceId=channel&deviceModel=web&deviceVersion=1.0&appVersion=1.0&deviceType=rokuChannel&deviceMake=rokuChannel&deviceDNT=1&advertisingId=channel&embedPartner=rokuChannel&appName=rokuchannel&is_lat=1&bmodel=bm1&content=channel&platform=web&tags=ROKU_CONTENT_TAGS&coppa=false&content_type=livefeed&rdid=channel&genre=ROKU_ADS_CONTENT_GENRE&content_rating=ROKU_ADS_CONTENT_RATING&studio_id=viacom&channel_id=channel
-#EXTINF:-1 tvg-id="80sRewind.us" tvg-name="80s Rewind" tvg-country="US" tvg-language="English" tvg-logo="http://images.pluto.tv/channels/5ca525b650be2571e3943c63/colorLogoPNG.png" group-title="Movies",Pluto TV 80s Rewind
+#EXTINF:-1 tvg-id="PlutoTV80sRewind.us" tvg-name="80s Rewind" tvg-country="US" tvg-language="English" tvg-logo="http://images.pluto.tv/channels/5ca525b650be2571e3943c63/colorLogoPNG.png" group-title="Movies",Pluto TV 80s Rewind
```
Comment (LaneSh4d0w) [inline channels/us.m3u:3140]: Incoherency in tvg-name ?

File: channels/us.m3u [hunk 6]
```diff
@@ -3119,19 +3129,19 @@ https://playerstv-samsungus.amagi.tv/playlist.m3u8
...
 https://service-stitcher.clusters.pluto.tv/v1/stitch/embed/hls/channel/5f4d86f519358a00072b978e/master.m3u8?deviceId=channel&deviceModel=web&deviceVersion=1.0&appVersion=1.0&deviceType=rokuChannel&deviceMake=rokuChannel&deviceDNT=1&advertisingId=channel&embedPartner=rokuChannel&appName=rokuchannel&is_lat=1&bmodel=bm1&content=channel&platform=web&tags=ROKU_CONTENT_TAGS&coppa=false&content_type=livefeed&rdid=channel&genre=ROKU_ADS_CONTENT_GENRE&content_rating=ROKU_ADS_CONTENT_RATING&studio_id=viacom&channel_id=channel
-#EXTINF:-1 tvg-id="AvidamodernadeRocko.us" tvg-name="A vida moderna de Rocko" tvg-country="US" tvg-language="Spanish" tvg-logo="https://images.pluto.tv/channels/5f6df6293a12e10007017396/colorLogoPNG.png" group-title="",Pluto TV A vida moderna de Rocko
+#EXTINF:-1 tvg-id="PlutoTVAvidamodernadeRocko.us" tvg-name="A vida moderna de Rocko" tvg-country="US" tvg-language="Spanish" tvg-logo="https://images.pluto.tv/channels/5f6df6293a12e10007017396/colorLogoPNG.png" group-title="",Pluto TV A vida moderna de Rocko
```
Comment (LaneSh4d0w) [inline channels/us.m3u:3144]: Incoherency in tvg-name ?

File: channels/us.m3u [hunk 7]
```diff
@@ -3723,139 +3733,139 @@ https://service-stitcher.clusters.pluto.tv/v1/stitch/embed/hls/channel/5f24662be
...
 https://service-stitcher.clusters.pluto.tv/v1/stitch/embed/hls/channel/5c362ded581a86051df509b4/master.m3u8?deviceId=channel&deviceModel=web&deviceVersion=1.0&appVersion=1.0&deviceType=rokuChannel&deviceMake=rokuChannel&deviceDNT=1&advertisingId=channel&embedPartner=rokuChannel&appName=rokuchannel&is_lat=1&bmodel=bm1&content=channel&platform=web&tags=ROKU_CONTENT_TAGS&coppa=false&content_type=livefeed&rdid=channel&genre=ROKU_ADS_CONTENT_GENRE&content_rating=ROKU_ADS_CONTENT_RATING&studio_id=viacom&channel_id=channel
-#EXTINF:-1 tvg-id="FemmesDeLoi.us" tvg-name="Femmes De Loi" tvg-country="CH;FR" tvg-language="French" tvg-logo="https://images.pluto.tv/channels/5f8ed673cad35f0007651fd4/colorLogoPNG.png" group-title="",Pluto TV Femmes De Loi
+#EXTINF:-1 tvg-id="PlutoTVFemmesDeLoi.us" tvg-name="Pluto TV Femmes De Loi" tvg-country="CH;FR" tvg-language="French" tvg-logo="https://images.pluto.tv/channels/5f8ed673cad35f0007651fd4/colorLogoPNG.png" group-title="",Pluto TV Femmes De Loi
```
Comment (LaneSh4d0w) [inline channels/us.m3u:3796]: Incoherency in tvg-name (see the ones named below)

**Comment by jkemp814** [comment]
Also, I wanted to ask. The news channels that start with the station call sign, i.e. WYNZ are at the bottom of the playlist. They all are either ABC, CBS, FOX, or NBC. Should those be renamed to put them up around their other news coparts?

Say 'ABC 7 (WYNZ)'

**Comment by Dum4G** [comment]
I don't mind doing that change

**Comment by jkemp814** [comment]
It will help people find their local channel. Some folks don't use the search feature of their player or know the call sign of the local channel. So, after I finish Pluto TV, I will go to that next. I'll also include the city it's broadcasting from.

**Comment by jkemp814** [comment]
I have noticed a lot of them are using the call sign in the tvg-id tag. Is that the way all of them should be? Not the name or description in the tvg-id tag?

**Comment by Dum4G** [comment]
That's not according to our rules for now but if we get a complete and updated list of call signs it should eventually become a standard IMO

**Comment by LaneSh4d0w** [comment]
If we get an EPG it would eventually be using call signs, because searching on all NBC, CBS, PBS names it wouldn't be easy.

**Comment by jkemp814** [comment]
There are a dozen or so like that at the bottom of the list. I'll stay with the current format and change those to match to be consistent with the majority.

**Comment by jkemp814** [comment]
I'm using the EPG from https://github.com/iptv-org/epg it does pick up some channels in the playlist.

**Comment by LaneSh4d0w** [comment]
> 
> 
> There are a dozen or so like that at the bottom of the list. I'll stay with the current format and change those to match to be consistent with the majority.

Some even use extended call signs : 
image

**Comment by jkemp814** [comment]
Yup, they are listing the digital channel DT-1, I have seen up to five under one call sign.

**Review by Dum4G — APPROVED** [review]

File: channels/us.m3u [hunk 8]
```diff
@@ -2381,15 +2381,25 @@ https://mhz-samsung-linear.samsung.wurl.com/manifest/playlist.m3u8
...
 https://granicusliveus9-a.akamaihd.net/miamifl/G2076_003/playlist.m3u8
 #EXTINF:-1 tvg-id="MiamiTV.us" tvg-name="Miami TV" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/WlrcyMe.png" group-title="XXX",Miami TV (720p)
+https://59ec5453559f0.streamlock.net/miamitv/smil:miamitv/playlist.m3u8
+#EXTINF:-1 tvg-id="MiamiTV.us" tvg-name="Miami TV" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/WlrcyMe.png" group-title="XXX",Miami TV (720p)
 https://59ec5453559f0.streamlock.net/miamitv/smil:miamitvWEB/playlist.m3u8
-#EXTINF:-1 tvg-id="MiamiTVJenny2.us" tvg-name=""Miami TV México" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/WlrcyMe.png" group-title="XXX",Miami TV México
+#EXTINF:-1 tvg-id="MiamiTV.us" tvg-name="Miami TV" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/WlrcyMe.png" group-title="XXX",Miami TV (720p)
+https://59ec5453559f0.streamlock.net/miamitv/smil:miamitvROKU/playlist.m3u8
+#EXTINF:-1 tvg-id="MiamiTVMexico.us" tvg-name=""Miami TV México" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/WlrcyMe.png" group-title="XXX",Miami TV México
```
Comment (LaneSh4d0w) [inline channels/us.m3u:2389]:
```suggestion
#EXTINF:-1 tvg-id="MiamiTVMexico.us" tvg-name="Miami TV México" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/WlrcyMe.png" group-title="XXX",Miami TV México
```

File: channels/us.m3u [hunk 9]
```diff
@@ -2381,15 +2381,25 @@ https://mhz-samsung-linear.samsung.wurl.com/manifest/playlist.m3u8
...
 https://59ec5453559f0.streamlock.net/mexicotv/smil:miamitvmexico/playlist.m3u8
+#EXTINF:-1 tvg-id="MiamiTVMexico.us" tvg-name=""Miami TV México" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/WlrcyMe.png" group-title="XXX",Miami TV México
```
Comment (LaneSh4d0w) [inline channels/us.m3u:2391]:
```suggestion
#EXTINF:-1 tvg-id="MiamiTVMexico.us" tvg-name="Miami TV México" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/WlrcyMe.png" group-title="XXX",Miami TV México
```

File: channels/us.m3u [hunk 10]
```diff
@@ -2381,15 +2381,25 @@ https://mhz-samsung-linear.samsung.wurl.com/manifest/playlist.m3u8
...
+https://59ec5453559f0.streamlock.net/mexicotv/smil:miamitvmexicoWEB/playlist.m3u8
+#EXTINF:-1 tvg-id="MiamiTVMexico.us" tvg-name=""Miami TV México" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/WlrcyMe.png" group-title="XXX",Miami TV México
```
Comment (LaneSh4d0w) [inline channels/us.m3u:2393]:
```suggestion
#EXTINF:-1 tvg-id="MiamiTVMexico.us" tvg-name="Miami TV México" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/WlrcyMe.png" group-title="XXX",Miami TV México
```

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**